flow360/services.py

```diff
@@ -61,45 +201,27 @@ def get_default_fork(params_as_dict):
     return params
 
 
-def validate_flow360_params_model(params_as_dict):
+def validate_flow360_params_model(params_as_dict, unit_system_name):
     """
     Validate a params dict against the pydantic model
     """
+
+    unit_system = init_unit_system(unit_system_name)
+
+    # removing _add properties as these are only used in WebUI
+    params_as_dict = remove_properties_with_prefix(params_as_dict, "_add")
```

Comment:
Removing the prefixed values here is okay, but during fork/retry won't we sometimes also have to handle those possibly? If we did, we probably don't want to delete them.